web/src/components/MemoEditor/Editor/index.tsx

```diff
@@ -194,7 +198,7 @@ const Editor = forwardRef(function Editor(props: Props, ref: React.ForwardedRef<
         insertText += `${Number(number) + 1}. `;
       }
       if (insertText) {
-        editorActions.insertText(insertText);
+        editorActions.insertText("\n" + insertText);
```

Comment:
Is `preventDefault` supposed to be here?

```suggestion
        event.preventDefault();
        editorActions.insertText("\n" + insertText);
```

Reply:
Good catch, actually missed this from some of my experiments. The reason we want to have `event.preventDefault` at the top of the function is so that we prevent the default browser behaviour before any async operations otherwise, we will get the aforementioned race condition. 

Pushed a change that simply removes the `if(insertText)` and always add the newline and insertText. That way, a newline will be inserted even if there are no insertText to insert.

Thanks for the review. Also really appreciate the work on memos, really cool program.